service/src/main/java/org/jboss/aerogear/unifiedpush/service/impl/ClientInstallationServiceImpl.java

```diff
@@ -83,8 +83,9 @@ public void addInstallation(Variant variant, Installation entity) {
 
     private void mergeCategories(Installation entity) {
```

Comment:
and perhaps a bit on why, not just what

On Mon, Jan 19, 2015 at 11:42 AM, Sebastien Blanc <notifications@github.com>
wrote:

> In
> service/src/main/java/org/jboss/aerogear/unifiedpush/service/impl/ClientInstallationServiceImpl.java
> <https://github.com/aerogear/aerogear-unifiedpush-server/pull/468#discussion-diff-23152575>
> :
>
> > @@ -83,8 +83,9 @@ public void addInstallation(Variant variant, Installation entity) {
> >
> >      private void mergeCategories(Installation entity) {
>
> We could move the comment that is inside the method //replace json
> deserialised categories with their persistent counter parts see
> Category.equals to the javadoc level, no ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/aerogear/aerogear-unifiedpush-server/pull/468/files#r23152575>
> .
>



-- 
Matthias Wessendorf

blog: http://matthiaswessendorf.wordpress.com/
sessions: http://www.slideshare.net/mwessendorf
twitter: http://twitter.com/mwessendorf